### PR TITLE
Content for beta assignment 5

### PR DESCRIPTION
**Pull request by praveek**

### CODE REVIEW

**Review by stcheng-adobe — COMMENTED** [review]
I think we should mention Android implementation is under development, but should be available soon.

File: beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+MobileCore.registerExtensions([Lifecycle.self, Identity.self, Signal.self, Edge.self, Analytics.self], {
+    // use the App id assigned to this application via Adobe Launch
+    MobileCore.configureWith(appId: self.LAUNCH_ENVIRONMENT_FILE_ID)
+})
+```
+
+2. Now run the app, find the `Track Action` or `Track State` button in the `Core` tab and click on it. You should see in the logs that a corresponding edge request is sent containing the track data. 
+
+3. You should see the reports get populated in the configured report suite 
+
```
Comment (stcheng-adobe) [inline beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md:77]: Should we include a sample screenshot of our testing report suite for customer who is not too familiar with AA?

File: beta/experience-platform-extension/tutorials/README.md [hunk 2]
```diff
@@ -80,4 +80,19 @@ Get started with [Assignment 4 - Rules for XDM events](https://aep-sdks.gitbook.
...
+
+### Assignment 5 (week Nov 17)
+
+In this assignment you will learn how to:
+
+1. Configure Analytics in Experience Edge environments
+2. Include Analytics Edge extension in sample apps
+3. Review trackAction and trackState calls​
+
+Get started with [Assignment 5 - Analytics edge extension](./tutorial-5-analytics-edge-extension.md).
```
Comment (emdobrin) [inline beta/experience-platform-extension/tutorials/README.md:93]: Make sure you add a link to this page in the SUMMARY.md file

File: beta/experience-platform-extension/tutorials/README.md [hunk 3]
```diff
@@ -80,4 +80,19 @@ Get started with [Assignment 4 - Rules for XDM events](https://aep-sdks.gitbook.
...
 * [ ] Ease to validate rules execution when using the AEP Mobile SDK
-* [ ] What validation techniques do you use when configuring rules remotely
\ No newline at end of file
+* [ ] What validation techniques do you use when configuring rules remotely
+
+### Assignment 5 (week Nov 17)
+
+In this assignment you will learn how to:
+
+1. Configure Analytics in Experience Edge environments
```
Comment (emdobrin) [inline beta/experience-platform-extension/tutorials/README.md:89]: Configure Analytics using the Edge Configuration.

File: beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+* Access to Adobe Experience Platform
+* Minimal Swift / Android development knowledge 
+* Knowledge about the AEP Edge extension
+* Completion of [Assignment 1 - AEP Edge extension setup and XDM usage](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-1-edge-extension-setup) (required to have a mobile property in Launch with Edge configuration setup).
+
+### Enable Analytics in Edge configuration
+
+In Adobe Experience Platform Launch, navigate to your mobile property and select Edge Configurations from the left panel, then select the configuration created in Assignment 1
+
+* Select development environment, enable `Adobe Analytics` and add development report suite to send analytics data.
```
Comment (emdobrin) [inline beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md:18]: and add a development report suite. If you don't have one, create a new report suite for this exercise using the Mobile template.

File: beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+### Set up the required fields
+
+In [Adobe Experience Platform Launch](https://experience.adobe.com/launch), go to the **Environments** tab in the mobile property created in Assignment 1 and click on the Development![img](https://firebasestorage.googleapis.com/v0/b/gitbook-28427.appspot.com/o/assets%2F-Lf1Mc1caFdNCK_mBwhe%2F-Lf1N06T8hdv0-r5jPPN%2F-Lf1N3-ofPO9fLFT1edw%2Fscreen-shot-2018-10-18-at-11.22.17-am.png?generation=1558039279051937&alt=media)icon. Find the Environment File ID at the top and copy it.
+
+Set the `LAUNCH_ENVIRONMENT_FILE_ID` to the copied Environment File ID in the `AppDelegate` \(iOS\) file.
+
+### Install the AnalyticsEdge Extension
+
+1. Add the following code to the `Podfile`
```
Comment (emdobrin) [inline beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md:49]: Let's provide the full path here sampleAppName/Swift/Podfile

File: beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+```text
+  pod 'AEPAnalyticsEdge', :git => 'https://github.com/adobe/aepsdk-analyticsedge-ios', :branch => 'main'
+```
+
+2. Run `pod install` in the `AEPSampleApp` base folder, it should download all the dependencies and add them to the project.
+3. Open the project from \`AEPSampleApp.xcworkspace
```
Comment (emdobrin) [inline beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md:56]: Same here

File: beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+// init SDK
+MobileCore.registerExtensions([Lifecycle.self, Identity.self, Signal.self, Edge.self, Analytics.self], {
+    // use the App id assigned to this application via Adobe Launch
+    MobileCore.configureWith(appId: self.LAUNCH_ENVIRONMENT_FILE_ID)
+})
+```
+
+2. Now run the app, find the `Track Action` or `Track State` button in the `Core` tab and click on it. You should see in the logs that a corresponding edge request is sent containing the track data. 
+
+3. You should see the reports get populated in the configured report suite
```
Comment (emdobrin) [inline beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md]: Let's end all the sentences with a period for consistency.

File: beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md (hunk 1 above)
Comment (emdobrin) [inline beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md:85]: Do we want to add some steps to inspect the events in Project Griffon?

File: beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+    // use the App id assigned to this application via Adobe Launch
+    MobileCore.configureWith(appId: self.LAUNCH_ENVIRONMENT_FILE_ID)
+})
+```
+
+2. Now run the app, find the `Track Action` or `Track State` button in the `Core` tab and click on it. You should see in the logs that a corresponding edge request is sent containing the track data. 
+
+3. You should see the reports get populated in the configured report suite. 
+
+Here is a sample report which captures actions and page views from the sample application.
```
Comment (emdobrin) [inline beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md:82]: a sample real-time Analytics report ...

**Review by stcheng-adobe — APPROVED** [review]

File: beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+#### Swift
+
+Download the iOS Swift Sample application from [https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-5](https://github.com/adobe/aepsdk-sample-app-ios/archive/beta-assignment-5.zip).
+
+To get started, follow the steps described in [AEP SDK Sample App Swift - Installation](https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-5#installation).
+{% endtab %}
+{% endtabs %}
+
+### Set up the required fields
+
```
Comment (cacheung) [inline beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md:38]: This part can be done after Install the AnalyticEdge Extension section.  But before Init Extension.

**Review by cacheung — COMMENTED** [review]
I just saw the comments were in pending.

**Pull request merged**